modules/test-it/src/test/scala/org/knora/webapi/responders/v2/ValuesEraseSpec.scala

```diff
@@ -572,6 +583,25 @@ final case class TestHelper(
                 .map(_.toMap)
   } yield ResourceWithValues(created, values)
 
+  def getUpdatedResource(resourceIri: ResourceIri): Task[ActiveResource] = for {
```

Comment:
Following the naming scheme in the TestHelper I would suggest to call this `findActiveResource`.

The method body completely ignores the fact that the resource was "updated" at some point.